src/encoded/tests/data/inserts/file.json

```
@@ -4770,5 +4770,41 @@
         "file_size": 370148294,
         "output_type": "regulatory elements prediction model",
         "lab": "/labs/richard-myers/"
+    },
+    {
+        "accession": "ENCFF910GSC",
+        "aliases": [
+            "erez-lieberman:HIC036-genome-subcompartments"
+        ],
+        "assembly": "hg19",
+        "award": "U54HG006998",
+        "dataset": "ENCSR000AJK",
+        "file_format": "bigWig",
```

The SLICE subcompartment outputs are bed files, maybe we could change the format here to be more consistent with what will end up on the portal?
```suggestion
        "file_format": "bed",
        "file_format_type": "bed9",
```